### ISSUE TITLE
brushnet is a super easy tool to use, can you do a flux version?

### ISSUE THREAD

**Issue by chaorenai** [nullquant/ComfyUI-BrushNet#133]
brushnet is a super easy tool to use, can you do a flux version?
brushnet是一个超级好用得工具，会做flux版本得嘛？

**Comment by nullquant** [comment]
Yeah, Flux is good, but it has completely different architecture. [The question](https://github.com/TencentARC/BrushNet/issues/63) was already asked.
